Tests/FunctionalTests/TestApp/Tasks/Winston.js

```diff
@@ -3,14 +3,14 @@ winston.level = 'silly';
 
 function logFn(type) {
     return (callback) => {
-        winston.log(type, "test %s", type);
+        winston.log(type, `test ${type}`);
         callback();
     }
 }
 
 function logFn2(type) {
     return (callback) => {
-        winston[type]("test %s", type);
+        winston[type](`test ${type}`);
```

Comment:
winston3 does not support string formatters like winston2 does, but winston3 does not support <node8. Sticking with winston2 for now, so this change probably isn't necessary.